round-2.html

```diff
@@ -4,9 +4,71 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="main.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@100&display=swap" rel="stylesheet">
   <title>Jeopardy</title>
 </head>
 <body>
-  <h1>Double Jeopardy</h1>
+  <h1 id="title">Double Jeopardy</h1>
```

Comment:
Everything from Round-1 applies. COMMENTS, looks good though and cool popup functionality